EDSMResponder/ConfigurationWindow.xaml.cs

```diff
@@ -45,10 +46,12 @@ private void updateEdsmConfiguration()
             if (!string.IsNullOrWhiteSpace(edsmApiKeyTextBox.Text))
             {
                 edsmConfiguration.apiKey = edsmApiKeyTextBox.Text.Trim();
```

Comment:
Turns out we want a different approach, but as a learning point it's best to use a local string variable to store the raw string, so that only the converted string ever hits the edsmConfiguration object.

```cs
string rawText = edsmApiKeyTextBox.Text.Trim();
edsmConfiguration.apiKey = Strings.JSONify(rawText);
```

Reply:
Noted. I imagine we may wish to take the same approach with ship names and similar then?